pkg/plugins/config/config.go

```diff
@@ -15,8 +15,9 @@ type Cfg struct {
 
 	PluginsPath string
 
-	PluginSettings       setting.PluginSettings
-	PluginsAllowUnsigned []string
+	PluginSettings               setting.PluginSettings
+	PluginsAllowUnsigned         []string
+	PluginForcePublicKeyDownload bool
```

Comment:
Considering this config is for a Grafana-specific implementation, I think it's probably better to keep this out of this and just depend on `*setting.Cfg` in pkg/services/pluginsintegration/keyretriever/dynamic/dynamic_retriever.go